test/run_pass/test_mqtt_sub_pub.js

```diff
@@ -28,7 +28,9 @@ Promise.all([
   babeee.subscribe('u/love', common.mustCall(function() {
     console.log('babeee subscribed u/love');
     setTimeout(function() {
-      yorkie.publish('u/love', 'endless');
+      yorkie.publish('u/love', 'endless', common.mustCall(function() {
+        console.log('yorkie published u/love');
+      }));
```

Comment:
What do you wanna test by this change?

Reply:
I check the case when options is undefined and the callback is the third argument, may it is not very rigorous, I will revise it.